### PR TITLE
WT-5041 Test calc modify test is not properly distributing the modifications

### PR DESCRIPTION
**Pull request by andym03**
Changed the array accessing logic as requested. Had to change (potentially fixed, honestly) the conditional logic to be an `or` instead of an `and`, as accessing the diff properly was impossible when i was 0.

### CODE REVIEW

File: test/suite/test_calc_modify.py [hunk 1]
```diff
@@ -83,10 +83,10 @@ def one_test(self, c, k, oldsz, repeatsz, nmod, maxdiff):
         orig = oldv
         newv = '' if self.valuefmt == 'S' else b''
         for i in range(nmod):
-            if i > 0 and offsets[i] - offsets[i - 1] < maxdiff:
+            if i > 0 or offsets[i] - offsets[i - 1] < maxdiff:
```
Comment (boboeee) [inline test/suite/test_calc_modify.py]: Should the _and_ be changed to _or_? it seems like if any value _i > 0_ will cause the continue branch constantly. Not sure if that was intended?

Comment (andym03) [reply test/suite/test_calc_modify.py]: You're right, the fix should have been making i > 1. Thanks!

File: test/suite/test_calc_modify.py [hunk 2]
```diff
@@ -82,11 +82,11 @@ def one_test(self, c, k, oldsz, repeatsz, nmod, maxdiff):
 
         orig = oldv
         newv = '' if self.valuefmt == 'S' else b''
-        for i in range(nmod):
+        for i in range(1, nmod):
             if i > 0 and offsets[i] - offsets[i - 1] < maxdiff:
```
Comment (boboeee) [inline test/suite/test_calc_modify.py]: Nitpick: i > 0 can be removed from if statement

Comment (boboeee) [inline test/suite/test_calc_modify.py]: Nitpick: i > 0 is not needed in the statement anymore

**Comment by boboeee** [comment]
LGTM.

**Pull request merged**